StaticAnalyzer/views/ios/common_rules.py

```diff
@@ -77,13 +77,14 @@
                  'Sensitive Information should be encrypted.'),
         'type': 'string',
         'string1': 'sqlite3_exec',
+        'string2': 'sqlite3_finalize',
         'level': 'info',
-        'match': 'single_string',
+        'match': 'string_or',
         'input_case': 'exact',
         'cvss': 0,
         'cwe': '',
         'owasp': '',
-        'owasp-mstg': OWASP_MSTG['storage-2'],
+        'owasp-mstg': OWASP_MSTG['storage-14'],
```

Comment:
Why not `storage-14` ? Isn't this similar to `[realm transactionWithBlock:` check?

Reply:
Missed that, I changed storage-2 to storage-14 in `[realm transactionWithBlock:` rule